kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -645,6 +645,23 @@ object KyuubiConf {
       .stringConf
       .createWithDefault("uid")
 
+  val AUTHENTICATION_LDAP_PASSWORD: OptionalConfigEntry[String] =
+    buildConf("kyuubi.authentication.ldap.bindnpw")
+      .doc("The password of the LDAP bind user " +
+        "is used to search for the full name of the authenticated user.")
+      .version("1.6.0")
+      .stringConf
+      .createOptional
```

Comment:
add it into `serverOnlyConfEntries`, otherwise, it will be visible in engine side